backend/src/main/java/springboot/DTO/CartDTO.java

```diff
@@ -0,0 +1,61 @@
+package springboot.DTO;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import springboot.model.CartItem;
+
+import java.util.List;
+
+public class CartDTO {
+
+    @JsonProperty("id")
+    private Long id;
+
+    @JsonProperty("owner")
+    private String owner;
+
+    @JsonProperty("items")
+    private List<ItemDTO> items;
```

Comment:
You might need to do the JsonIgnore tag on this so we don't get recursive results when we return.